tests/test_base_env.py

```diff
@@ -87,7 +91,7 @@ def test_tabular_env_validation():
         transition_matrix=np.zeros((3, 1, 3)),
         reward_matrix=np.zeros((3,)),
     )
-    env.reset()
+    env.reset(seed=0)
```

Comment:
Why do we want to seed this now but not before?

Reply:
With the new API, the first `reset()` needs a seed. Otherwise we can't do any steps and possibly not even sample an initial state.